src/ansys/tools/report.py

```diff
@@ -15,7 +15,7 @@
 
 __version__ = importlib_metadata.version("pyansys-tools-report")
 
-__ANSYS_VARS__ = ("AWP_ROOT", "ANS", "FLUENT", "MAPDL", "AEDT", "DPF")
+__ANSYS_VARS_PREFIX__ = ("AWP", "ACP", "ANS", "FLUENT", "MAPDL", "AEDT", "DPF")
```

Comment:
I don't think we have an "AEDT" environment variable, it is `ANSYSEM_ROOT`.
However we do have `SIMPLORER_HOME `and `SIWAVE_INSTALL_DIR`.
```suggestion
__ANSYS_VARS_PREFIX__ = ("AWP", "ACP", "ANS", "FLUENT", "MAPDL", "DPF", "SIMPLORER", "SIWAVE")
```

Reply:
Thanks for the heads up @MaxJPRey